### PR TITLE
WIP tests: addition of test cases for sentry

### PR DESCRIPTION
**Pull request by bruno314**

### CODE REVIEW

File: AUTHORS.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    waive the privileges and immunities granted to it by virtue of its status
+    as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+
+Authors
+=======
+
+Module providing logging capabilities
+
+- CERN <info@invenio-software.org>
```
Comment (jirikuncar) [inline AUTHORS.rst]: please update with your name and remove CERN.

File: invenio_logging/sentry.py [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+from raven.contrib.flask import Sentry
+from raven.handlers.logging import SentryHandler
+from raven.processors import SanitizePasswordsProcessor
+from werkzeug.local import LocalProxy
+
+from flask_babelex import gettext as _
+
+
+def fakecapture(self, *args, **kwargs):
+    print ("@@" + str(args) + str(kwargs))
```
Comment (jirikuncar) [inline invenio_logging/sentry.py]: is it still needed?

File: invenio_logging/views.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
```
Comment (jirikuncar) [inline invenio_logging/views.py]: please remove this file

File: requirements-devel.txt [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+#
+# TODO: Add development versions of some important dependencies here to get a
```
Comment (jirikuncar) [inline requirements-devel.txt]: add links to required packages

File: setup.py [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+with open(os.path.join('invenio_logging', 'version.py'), 'rt') as fp:
+    exec(fp.read(), g)
+    version = g['__version__']
+
+setup(
+    name='invenio-logging',
+    version=version,
+    description=__doc__,
+    long_description=readme + '\n\n' + history,
+    keywords='invenio TODO',
```
Comment (kiryx) [inline setup.py]: TODO keywords

Comment (bruno314) [inline setup.py]: OK

**Comment by otron** [comment]
Besides the still-remaining commented out bits of code and TODOs it looks good!

File: invenio_logging/fs.py [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+
+from flask_babelex import gettext as _
+
+
+class InvenioLoggingFs(object):
+    """Invenio-Logging extension. Filesystem handler."""
+
+    def __init__(self, app=None):
+        """Extension initialization."""
+        _('A translation string')
```
Comment (lnielsen) [inline invenio_logging/fs.py]: Please remove this line

File: invenio_logging/fs.py [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+        app.config.setdefault(
+            'LOGGING_FS_LEVEL',
+            'DEBUG' if app.debug else 'WARNING'
+        )
+
+        # Create log directory if it does not exists
+        try:
+            os.makedirs(
+                os.path.join(app.instance_path,
+                             app.config.get('CFG_LOGDIR', ''))
```
Comment (lnielsen) [inline invenio_logging/fs.py]: CFG_LOGDIR -> LOGGING_FS_LOGDIR

File: invenio_logging/fs.py [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+            )
+        except Exception:
+            pass
+
+        handler = RotatingFileHandler(
+            os.path.join(
+                app.instance_path,
+                app.config.get('CFG_LOGDIR', ''),
```
Comment (lnielsen) [inline invenio_logging/fs.py]: ditto

**Pull request closed without merging**